app/client/ui-modules/guest-landing-page/GuestLandingPage.tsx

```diff
@@ -1,64 +1,71 @@
-import React from "react";
-import { GuestLandingPageUiState } from "./state/GuestLandingPageUiState";
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { GuestLandingPageUiState} from "./state/GuestLandingPageUiState";
 import { Link } from "react-router";
 import Ripple from "./animations/Ripple";
-import { selectGuestLandingPageUiState } from "./state/reducers/guest-landing-page-slice";
-import { useSelector } from "react-redux";
 import { SideNavBar } from "../navigation-bars/side-nav-bars/SideNavbar";
 import { TopNavbar } from "../navigation-bars/TopNavbar";
```

Comment:
I have refactored this whole page so no need for manual add in each page. but its okay if we can getyour changes in im happy to override.
thats just FYI